### ISSUE TITLE
Crash when opening preferences

### ISSUE THREAD

**Issue by Sirush** [rampaa/JL#23]
Hello,
I just downloaded the latest release for the first time and it crashes when I'm trying to open the preferences (both x86 and x64 versions).

Here's the full stack trace

```
2022-10-03 16:06:01.968 +02:00 [ERR] System.ArgumentException: 'NaN' is not a valid value for property 'Value'.
   at System.Windows.DependencyObject.SetValueCommon(DependencyProperty dp, Object value, PropertyMetadata metadata, Boolean coerceWithDeferredReference, Boolean coerceWithCurrentValue, OperationType operationType, Boolean isInternal)
   at System.Windows.DependencyObject.SetValue(DependencyProperty dp, Object value)
   at HandyControl.Controls.NumericUpDown.set_Value(Double value)
   at JL.Windows.ConfigManager.LoadPreferences(PreferencesWindow preferenceWindow) in D:\a\JL\JL\JL.Windows\ConfigManager.cs:line 741
   at JL.Windows.Utilities.WindowsUtils.ShowPreferencesWindow() in D:\a\JL\JL\JL.Windows\Utilities\WindowsUtils.cs:line 206
   at JL.Windows.GUI.MainWindow.ShowPreferences(Object sender, RoutedEventArgs e) in D:\a\JL\JL\JL.Windows\GUI\MainWindow.xaml.cs:line 572
   at System.Windows.RoutedEventHandlerInfo.InvokeHandler(Object target, RoutedEventArgs routedEventArgs)
   at System.Windows.EventRoute.InvokeHandlersImpl(Object source, RoutedEventArgs args, Boolean reRaised)
   at System.Windows.UIElement.RaiseEventImpl(DependencyObject sender, RoutedEventArgs args)
   at System.Windows.UIElement.RaiseEvent(RoutedEventArgs e)
   at System.Windows.Controls.MenuItem.InvokeClickAfterRender(Object arg)
   at System.Windows.Threading.ExceptionWrapper.InternalRealCall(Delegate callback, Object args, Int32 numArgs)
   at System.Windows.Threading.ExceptionWrapper.TryCatchWhen(Object source, Delegate callback, Object args, Int32 numArgs, Delegate catchHandler)
   at System.Windows.Threading.DispatcherOperation.InvokeImpl()
   at System.Windows.Threading.DispatcherOperation.InvokeInSecurityContext(Object state)
   at MS.Internal.CulturePreservingExecutionContext.CallbackWrapper(Object obj)
   at System.Threading.ExecutionContext.RunInternal(ExecutionContext executionContext, ContextCallback callback, Object state)
--- End of stack trace from previous location ---
   at System.Threading.ExecutionContext.RunInternal(ExecutionContext executionContext, ContextCallback callback, Object state)
   at System.Threading.ExecutionContext.Run(ExecutionContext executionContext, ContextCallback callback, Object state)
   at MS.Internal.CulturePreservingExecutionContext.Run(CulturePreservingExecutionContext executionContext, ContextCallback callback, Object state)
   at System.Windows.Threading.DispatcherOperation.Invoke()
   at System.Windows.Threading.Dispatcher.ProcessQueue()
   at System.Windows.Threading.Dispatcher.WndProcHook(IntPtr hwnd, Int32 msg, IntPtr wParam, IntPtr lParam, Boolean& handled)
   at MS.Win32.HwndWrapper.WndProc(IntPtr hwnd, Int32 msg, IntPtr wParam, IntPtr lParam, Boolean& handled)
   at MS.Win32.HwndSubclass.DispatcherCallbackOperation(Object o)
   at System.Windows.Threading.ExceptionWrapper.InternalRealCall(Delegate callback, Object args, Int32 numArgs)
   at System.Windows.Threading.ExceptionWrapper.TryCatchWhen(Object source, Delegate callback, Object args, Int32 numArgs, Delegate catchHandler)
   at System.Windows.Threading.Dispatcher.LegacyInvokeImpl(DispatcherPriority priority, TimeSpan timeout, Delegate method, Object args, Int32 numArgs)
   at MS.Win32.HwndSubclass.SubclassWndProc(IntPtr hwnd, Int32 msg, IntPtr wParam, IntPtr lParam)
   at MS.Win32.UnsafeNativeMethods.DispatchMessage(MSG& msg)
   at System.Windows.Threading.Dispatcher.PushFrameImpl(DispatcherFrame frame)
   at System.Windows.Threading.Dispatcher.PushFrame(DispatcherFrame frame)
   at System.Windows.Threading.Dispatcher.Run()
   at System.Windows.Application.RunDispatcher(Object ignore)
   at System.Windows.Application.RunInternal(Window window)
   at System.Windows.Application.Run()
   at JL.Windows.App.Main()```

**Comment by rampaa** [comment]
Hi,

Can you share the content of your "JL.dll.config" with me? Also what OS do you use?

**Comment by Sirush** [comment]
Here it is,
I'm on windows 10 Pro 21H2, using English (UK) Locale and Japanese for non-unicode
```<?xml version="1.0" encoding="utf-8"?>
<configuration>
    <appSettings>
        <add key="AnkiConnectUri" value="http://localhost:8765" />
        <add key="SearchUrl" value="https://www.google.com/search?q={SearchTerm}&amp;hl=ja" />
        <add key="HighlightLongestMatch" value="False" />
        <add key="AutoPlayAudio" value="False" />
        <add key="Precaching" value="False" />
        <add key="CheckForJLUpdatesOnStartUp" value="True" />
        <add key="AlwaysOnTop" value="True" />
        <add key="RequireLookupKeyPress" value="False" />
        <add key="DisableHotkeys" value="False" />
        <add key="Focusable" value="True" />
        <add key="AnkiIntegration" value="False" />
        <add key="MaxSearchLength" value="37" />
        <add key="KanjiMode" value="False" />
        <add key="ForceSyncAnki" value="False" />
        <add key="AllowDuplicateCards" value="False" />
        <add key="LookupRate" value="0" />
        <add key="MainWindowTextColor" value="#FFFFFFFF" />
        <add key="MainWindowBacklogTextColor" value="#FFFFE4C4" />
        <add key="PrimarySpellingColor" value="#FFD2691E" />
        <add key="ReadingsColor" value="#FFDAA520" />
        <add key="AlternativeSpellingsColor" value="#FFFFFFFF" />
        <add key="DefinitionsColor" value="#FFFFFFFF" />
        <add key="FrequencyColor" value="#FFFFFFFF" />
        <add key="DeconjugationInfoColor" value="#FFFFFFFF" />
        <add key="SeparatorColor" value="#FFFFFFFF" />
        <add key="DictTypeColor" value="#FFADD8E6" />
        <add key="HighlightColor" value="#FFF0F8FF" />
        <add key="PopupBackgroundColor" value="#FF000000" />
        <add key="PopupOpacity" value="70" />
        <add key="PrimarySpellingFontSize" value="21" />
        <add key="ReadingsFontSize" value="19" />
        <add key="AlternativeSpellingsFontSize" value="17" />
        <add key="DefinitionsFontSize" value="17" />
        <add key="FrequencyFontSize" value="17" />
        <add key="DeconjugationInfoFontSize" value="17" />
        <add key="DictTypeFontSize" value="15" />
        <add key="MaxNumResultsNotInMiningMode" value="7" />
        <add key="PopupFocusOnLookup" value="False" />
        <add key="PopupXOffset" value="10" />
        <add key="PopupYOffset" value="20" />
        <add key="ShowMiningModeReminder" value="True" />
        <add key="DisableLookupsForNonJapaneseCharsInPopups" value="True" />
        <add key="PopupMaxWidth" value="700" />
        <add key="PopupMaxHeight" value="520" />
        <add key="FixedPopupPositioning" value="False" />
        <add key="FixedPopupXPosition" value="0" />
        <add key="FixedPopupYPosition" value="0" />
        <add key="PopupFlip" value="Both" />
        <add key="LookupMode" value="Hover" />
        <add key="DisableHotkeysKeyGesture" value="Pause" />
        <add key="MiningModeKeyGesture" value="M" />
        <add key="PlayAudioKeyGesture" value="P" />
        <add key="KanjiModeKeyGesture" value="K" />
        <add key="LookupKeyKeyGesture" value="LeftShift" />
        <add key="ShowManageDictionariesWindowKeyGesture" value="D" />
        <add key="ShowManageFrequenciesWindowKeyGesture" value="F" />
        <add key="ShowPreferencesWindowKeyGesture" value="L" />
        <add key="ShowAddNameWindowKeyGesture" value="N" />
        <add key="ShowAddWordWindowKeyGesture" value="W" />
        <add key="SearchWithBrowserKeyGesture" value="S" />
        <add key="MousePassThroughModeKeyGesture" value="T" />
        <add key="InvisibleToggleModeKeyGesture" value="I" />
        <add key="SteppedBacklogBackwardsKeyGesture" value="Left" />
        <add key="SteppedBacklogForwardsKeyGesture" value="Right" />
        <add key="InactiveLookupModeKeyGesture" value="Q" />
        <add key="MotivationKeyGesture" value="O" />
        <add key="ClosePopupKeyGesture" value="Escape" />
        <add key="ShowStatsKeyGesture" value="Y" />
        <add key="NextDictKeyGesture" value="Next" />
        <add key="PreviousDictKeyGesture" value="PageUp" />
        <add key="AlwaysOnTopKeyGesture" value="R" />
        <add key="TextOnlyVisibleOnHoverKeyGesture" value="E" />
        <add key="TextBoxIsReadOnlyKeyGesture" value="U" />
        <add key="CaptureTextFromClipboardKeyGesture" value="F12" />
        <add key="MainWindowOpacity" value="30" />
        <add key="MainWindowFontSize" value="50" />
        <add key="MainWindowFont" value="Meiryo" />
        <add key="MainWindowBackgroundColor" value="#FF000000" />
        <add key="TextOnlyVisibleOnHover" value="False" />
        <add key="ChangeMainWindowBackgroundOpacityOnUnhover" value="False" />
        <add key="MainWindowBackgroundOpacityOnUnhover" value="0.2" />
        <add key="TextBoxIsReadOnly" value="True" />
        <add key="TextBoxTrimWhiteSpaceCharacters" value="True" />
        <add key="TextBoxRemoveNewlines" value="False" />
        <add key="TextBoxApplyDropShadowEffect" value="True" />
        <add key="CaptureTextFromClipboard" value="True" />
        <add key="OnlyCaptureTextWithJapaneseCharsFromClipboard" value="True" />
        <add key="DisableLookupsForNonJapaneseCharsInMainWindow" value="False" />
        <add key="MainWindowDynamicHeight" value="False" />
        <add key="MainWindowDynamicWidth" value="False" />
        <add key="MainWindowHeight" value="50" />
        <add key="MainWindowWidth" value="100" />
        <add key="MainWindowMaxDynamicWidth" value="800" />
        <add key="MainWindowMaxDynamicHeight" value="269" />
        <add key="MainWindowTopPosition" value="286" />
        <add key="MainWindowLeftPosition" value="-1842" />
        <add key="PopupFont" value="Meiryo" />
        <add key="PopupDynamicHeight" value="True" />
        <add key="PopupDynamicWidth" value="True" />
    </appSettings>
</configuration>```

**Comment by rampaa** [comment]
> I tried going into preferences with anki opened, and now it works

This must be a coincidence.

The issue should be fixed in version 1.14.2.
